Notifications/NotificationService.swift

```diff
@@ -34,39 +36,39 @@ class NotificationService: UNNotificationServiceExtension {
             }
             return nil
         }
-        
+
         func exitGracefully(_ reason: String = "") {
             let bca = request.content.mutableCopy() as? UNMutableNotificationContent
             bca!.title = reason
             contentHandler(bca!)
         }
-        
+
         guard let content = (request.content.mutableCopy() as? UNMutableNotificationContent) else {
             return exitGracefully()
         }
-        
+
         let userInfo: [AnyHashable: Any] = request.content.userInfo
-        
+
         guard let attachmentURL = userInfo["media-url"] as? String else {
             return exitGracefully()
         }
-        
+
         guard let imageData  =
             try? Data(contentsOf: URL(string: attachmentURL)!)
             else {
                 return exitGracefully()
         }
-        
+
         guard let attachment =
             save("image.png", data: imageData, options: nil)
             else {
                 return exitGracefully()
         }
-        
+
         content.attachments = [attachment]
         contentHandler(content.copy() as! UNNotificationContent)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Notifications/NotificationService.swift#L69"></span>Force casts should be avoided.<br />
<code>force_cast</code> <code>NotificationService.swift:69</code></td>
    </tr>
  </tbody>
</table>